installer/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.2
+sbt.version=1.2.8
```

Comment:
it does not work with 1.3.x?

Reply:
I think it does. I remember my editor (Intellij) automatically updated this value. I'll revert this change and test again.